Dockerfile

```diff
@@ -19,11 +19,10 @@ RUN go mod download
 #
 COPY . .
 
-RUN go test ./...  && \
-    go install \
-        -ldflags "-w -X 'github.com/Pix4D/cogito/resource.buildinfo=$BUILD_INFO'" \
-        ./cmd/cogito \
-        ./cmd/out
+RUN go test ./cogito ./cmd/cogito
+RUN go install \
```

Comment:
There is no need to split; I did it because this is not the final image (see line 37 below).
Do you see negative consequences?

Reply:
No big deal. It might very slightly increase the final image size because it should record an additional layer but unless the `go test` does touch the filesystem it will be an empty one. (I used conditional because I don't know if it can optimize away and empty layer or not.)